### PR TITLE
2.0 preview

### PR DESCRIPTION
**Pull request by e111077**
DO NOT MERGE

No significant changes other than the var being updated to v1

### CODE REVIEW

File: paper-item.html [hunk 1]
```diff
@@ -88,14 +88,14 @@
...
+        @apply --layout-center;
+        @apply --paper-font-subhead;
 
-        @apply(--paper-item);
+        @apply --paper-item;
       }
     </style>
-
+    <slot></slot>
     <content></content>
```
Comment (keanulee) [inline paper-item.html]: @e111077 Can you remove the `<content>` as well? `paper-item` has some shared styles on the class `.paper-item`, which unfortunately clashes with how Shady DOM scopes styles when extraneous elements (like `<content>`) are present.

**Comment by notwaldorf** [comment]
@keanulee just butting in: for your app, I'd recommend using the shared style in `paper-styles/element-styles/paper-item.html` on a `<button>`, rather than `paper-item` the element :)

File: demo/index.html [hunk 2]
```diff
@@ -29,24 +29,26 @@
   <link rel="import" href="../paper-item.html">
   <link rel="import" href="../paper-item-body.html">
   <link rel="import" href="../../paper-styles/color.html">
+  
+  <custom-style include="demo-pages-shared-styles">
```
Comment (bicknellr) [inline demo/index.html]: I don't think you need the `include="..."` here as long as it's on the `<style>` below.

**Comment by bicknellr** [comment]
[This `<style is="custom-style">`](https://github.com/PolymerElements/paper-item/pull/94/files#diff-3e22c50371c76d64ea9738dbc46a6e57R70) needs to be wrapped.

File: demo/index.html (hunk 2 above)
Comment (e111077) [inline demo/index.html]: dorp

File: paper-item-body.html [hunk 3]
```diff
@@ -57,22 +57,24 @@
         min-height: var(--paper-item-body-three-line-min-height, 88px);
       }
 
-      :host > ::content > * {
+      :host > ::content > *,
```
Comment (bicknellr) [inline paper-item-body.html]: I think you can remove selectors with `::content`.

File: paper-item-body.html [hunk 4]
```diff
@@ -57,22 +57,24 @@
...
+      :host > ::slotted(*) {
         overflow: hidden;
         text-overflow: ellipsis;
         white-space: nowrap;
       }
 
-      :host > ::content [secondary] {
-        @apply(--paper-font-body1);
+      :host > ::content [secondary],
```
Comment (bicknellr) [inline paper-item-body.html]: (also here)

File: paper-item-body.html (hunk 3 above)
Comment (e111077) [inline paper-item-body.html]: 👍

File: paper-item-body.html (hunk 4 above)
Comment (e111077) [inline paper-item-body.html]: 🍻

File: demo/index.html [hunk 5]
```diff
@@ -84,25 +88,25 @@
       </template>
     </demo-snippet>
 
-    <h3>To add a leading element, use a paper-icon-item with an <i>item-icon</i> attribute. This
-    leading image can be an iron-icon, or any other regular element.</h3>
+    <h3>To add a leading element, use a paper-icon-item with an <i>item-icon</i> attribute or <code>slot="item-icon"</code>
+    attribute in webcomponents v1. This leading image can be an iron-icon, or any other regular element.</h3>
```
Comment (bicknellr) [inline demo/index.html]: Using the `item-icon` attribute doesn't work anymore, right? If so, we should update this comment and remove the attribute from items in the demos.

Comment (notwaldorf) [inline demo/index.html]: done!

**Comment by notwaldorf** [comment]
@bicknellr done! 👀 🙏 ?

**Pull request merged**